public/index.html

```diff
@@ -33,17 +33,16 @@
       el: '#debt-collective-header',
       data: {
         headerLinks: [
-          { text: 'Collectives', href: '/' },
-          { text: 'Campaigns', href: '#campaigns' },
-          { text: 'Events', href: '#events', onclick: 'window.loggit()' },
-        ],
-        dropdownLinks: [
+          { text: 'My Disputes', href: '#disputes' },
+          { text: 'Dispute Your Debt', href: '#debt' },
+          { text: 'Campaings', href: '#camp' },
+          { text: 'Events', href: '#events' },
           { text: 'News', href: '#news' },
           { text: 'Dispute Your Debt', href: '#dispute-your-debt', onclick: 'window.loggit()' },
           { text: 'Admin Link Only', href: '#dispute-your-asdf', onclick: 'window.loggit()', roles: ['dispute-admin'] },
           { text: 'About the Debt Collective', href: '#about' },
-          { text: 'Community Wiki', href: '#wiki' },
-          { text: 'Medicare For All', href: '#m4a' },
+        ],
+        dropdownLinks: [
```

Comment:
@duranmla we still want to control what links we can put here. Is that feature still working?

Reply:
So mainly the introduced approach is intended to display all nav links as long there is space for them, otherwise the dropdown will appear taking the items that overflow the screen, good thing is that we are ensuring that is the user has the space we won't "hide" anything that can be relevant for the user.